subsystem/identity_provider_resource_test.go

```diff
@@ -39,7 +39,43 @@ var _ = Describe("Identity provider creation", func() {
 				  "state": "ready"
 				}`),
 			),
+			CombineHandlers(
+				VerifyRequest(http.MethodGet, "/api/clusters_mgmt/v1/clusters/123"),
+				RespondWithJSON(http.StatusOK, `{
+				  "id": "123",
+				  "name": "my-cluster",
+				  "state": "ready"
+				}`),
+			),
+		)
+	})
+
+	It("Fail if cluster_id not found", func() {
+		// Prepare the server:
+		server.Reset()
```

Comment:
I prefer to use a context for the other test cases (and removing the before each)

Reply:
Done